src/main.rs

```diff
@@ -77,43 +77,34 @@ struct Args {
     flag_version: bool,
 }
 
-fn write_dotfile_to_disk(filepath: &str, arena: treediff::Arena<String, String>) {
-    let mut dotfile = match File::create(&filepath) {
-        Ok(f) => f,
-        Err(_) => {
-            writeln!(&mut stderr(), "Could not create file {}.", &filepath).ok();
+fn write_dotfile_to_disk(filepath: &str, arena: &ast::Arena<String, String>) {
+    match emitters::write_dotfile_to_disk(filepath, arena) {
```

Comment:
This code in main is very noisy, with all the matches on different error types. However, I'd like to leave this for another PR which would clean up the whole lot in one go.